crates/router/src/configs/settings.rs

```diff
@@ -39,6 +39,13 @@ pub enum Subcommand {
     GenerateOpenapiSpec,
 }
 
+#[cfg(feature = "kms")]
+/// Store the decrypted kms secret values for active use in the application
+#[derive(Clone)]
+pub struct ActiveKmsSecrets {
+    pub jwekey: masking::Secret<Jwekey>,
```

Comment:
StrongSecret so that it will be zero'ed out on drop.

Reply:
So, my thought process was on the performance side
> The `AppState` is cloned on every request and then dropped at the end. 
> Having it as `StrongSecret` will incur higher cost for every request.

So, what do you think, should we still go with `StrongSecret`?

Reply:
If you think the zero'ing out would be costly due to per request cloning, Go ahead with secret but put a note of the same in code.

Reply:
Done